fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingThrottle.java

```diff
@@ -223,6 +225,8 @@ <R> CompletableFuture<R> throttledRunAsync(@Nonnull final Function<FDBRecordStor
                         if (handleLessenWork != null) {
                             handleLessenWork.accept(fdbE, onlineIndexerLogMessageKeyValues);
                         }
+                        final ExponentialDelay delay = new ExponentialDelay(common.getRunner().getDatabase().getFactory().getInitialDelayMillis(),
+                                common.getRunner().getDatabase().getFactory().getMaxDelayMillis());
```

Comment:
Why was this moved?

Reply:
Not a big reason - just prefer locality when possible as a good coding habit (especially since it is unused in the common, no error path).

Reply:
So, does this need to be moved back?

Reply:
Moved back.